bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/LQ_cmutau/LQ_cmutau_proc_card.dat

```diff
@@ -0,0 +1,41 @@
+#************************************************************
+#*                     MadGraph5_aMC@NLO                    *
+#*                                                          *
+#*                *                       *                 *
+#*                  *        * *        *                   *
+#*                    * * * * 5 * * * *                     *
+#*                  *        * *        *                   *
+#*                *                       *                 *
+#*                                                          *
+#*                                                          *
+#*         VERSION 2.6.3.2               2018-06-22         *
+#*                                                          *
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#*                                                          *
+#*               Command File for MadGraph5_aMC@NLO         *
+#*                                                          *
+#*     run as ./bin/mg5_aMC  filename                       *
+#*                                                          *
+#************************************************************
+set default_unset_couplings 99
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set loop_color_flows False
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+import model sm
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+import model LeptoQ_UFO
+generate p p > t t~, t > b w+, t~ > c~ ta+ mu-
+add process p p > t t~, t > c ta- mu+, t~ > b~ w-
```

Comment:
Since you don't decay the W's explicitly here and don't use MadSpin, it means they will have to be decayed by Pythia. This means the spin correlation information will not be present.
Is that expected?

Reply:
To reply to the major question, I am not well understanding the generation process. Is a customize_card is necessary? I am just thinking naively that the extramodel we provided will contain all the parameters needed for the process. I would like to know what a customize_card or definition of the signal parameters does.

Yes, it is what we expected. We use inclusively W decay samples and are not considering the use of spin correlation.

Reply:
> We use inclusively W decay samples and are not considering the use of spin correlation.

Ok, thanks for clarifying.

Reply:
I thought I already pointed out W decay and its spin information in the internal meeting? As far as I remember, attaching madspin had some problems in this model. Can you make further tests?

Reply:
Can't you include the W decay in the madgraph process without using madspin?

Reply:
I can add the explicit W decay modes in the madgraph process.
What I want to ask is,
since there are two W decay modes, hadronic or leptonic, if we request MC samples, then are two samples produced, respectively?

Reply:
Isn't it possible?
define allw = u c d s u~ c~ d~ s~ ve vm vt e- mu- ve~ vm~ vt~ e+ mu+ t b t~ b~ ta- ta+
generate p p > t t~, (t > w+ b, w+ > allw allw), (t~ > w- b~, w- > allw allw)

Reply:
Just write `, w- > all all` and `, w+ > all all` to include everything in one sample.

Reply:
Then I will add w decay to "all" in our proc card and update. Thank you :)

Reply:
Then I will add w decay to "all" in our proc card and update. Thank you :)

Reply:
> Just write `, w- > all all` and `, w+ > all all` to include everything in one sample.

I think this will create un-intended diagrams such as w+ -> w+ h. (I see some warning in MG5) Is it ok to proceed? (Even though its cross section would be very small) For sure, in madspin, branching rations will be recalculated...